bigbluebutton-html5/imports/ui/components/modal-consent/container.jsx

```diff
@@ -0,0 +1,21 @@
+
+import { Meteor } from 'meteor/meteor';
+import { withTracker } from 'meteor/react-meteor-data';
+import ModalConsent from './component';
+import { RecordMeetings } from '/imports/api/meetings';
+import Meetings from '../../../api/meetings/index';
+
+
+let ModalConsentContainer = withTracker(()=>{
+    Meteor.subscribe('record-meetings');
+    Meteor.subscribe('breakouts');
```

Comment:
There's a better place to add this subscriptions in my opinion:

https://github.com/JoVictorNunes/bigbluebutton/blob/2e72946bc1f44383b3ddcae5c56cb4bba443e1b4/bigbluebutton-html5/imports/ui/components/subscriptions/component.jsx#L17-L23